mgmt/src/grpc/server.rs

```diff
@@ -673,6 +771,11 @@ impl BasicConfigManager {
                 _ => None,
             };
 
+            // Convert OSPF interface if present - FIXED
```

Comment:
What do you mean by “FIXED”?

Reply:
I believe @sergeymatov added this. It is anyway removed later.